crates/matrix-sdk-crypto/src/store/traits.rs

```diff
@@ -423,6 +429,14 @@ impl<T: CryptoStore> CryptoStore for EraseCryptoStoreError<T> {
     ) -> Result<bool, Self::Error> {
         self.0.try_take_leased_lock(lease_duration_ms, key, holder).await.map_err(Into::into)
     }
+
+    async fn save_next_batch_token(&self, token: Option<String>) -> Result<(), Self::Error> {
```

Comment:
Do we need this? If we do, can't we move this out of the trait and put it into the `Store`, since it can call `save_changes()` internally.

Reply:
Mostly for testing. I could probably put it in a side-method that's testing-only.

Reply:
I think putting it onto the `Store` then makes sense, we have a bunch of such methods there already.

Reply:
Uh, realized I could even just use `save_changes` from the tests, and remove `save_next_batch_token` :eyes: